lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -88,3 +88,7 @@ ecdsa==0.13
 
 # GenomeSpace dependencies
 python-genomespaceclient==0.1.8
+
+# ISA
+git+https://github.com/ISA-tools/isa-rwval.git@develop#egg=isa-rwval
```

Comment:
Galaxy dependencies are handled using pipenv so in order to add deps we need to modify https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/pipfiles/default/Pipfile and run the proper makefile target to rebuild this file - isntead of manual modification.

Also as mentioned on the starforge PR I think this should get to pypi first.

Reply:
Is the wheel ready? May I replace this line with `isa-rwval==0.10.0` and test it?